### PR TITLE
First bloom filter reachability map implementation

### PR DESCRIPTION
**Pull request by Newbrict**
No increment behavior yet.

### CODE REVIEW

File: simple_bloom_reachability_map.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+func NewSimpleBloomReachabilityMap() ReachabilityMap {
+	m := SimpleBloomReachabilityMap{f: bloom.New(1000, 4)}
+	return m
+}
+
+func (m SimpleBloomReachabilityMap) IsReachable(n NodeAddress) bool {
+	entry := []byte(n)
+	res := m.f.Test(entry)
```
Comment (c00w) [inline simple_bloom_reachability_map.go]: just do return m.f.Test

File: simple_bloom_reachability_map.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	return res
+}
+
+func (m SimpleBloomReachabilityMap) AddEntry(n NodeAddress) {
+	entry := []byte(n)
+	m.f.Add(entry)
```
Comment (c00w) [inline simple_bloom_reachability_map.go]: You can cast to []byte inline.

**Comment by c00w** [comment]
Just call it BloomReachabilityMap?

**Comment by c00w** [comment]
Otherwise LGTM

**Comment by Newbrict** [comment]
I didn't want to take up BloomReachabilityMap because I figured there would be different implementations of bloom filter reachabililty maps, unless your thinking is that we just keep evolving this one?

File: simple_bloom_reachability_map.go (hunk 2 above)
Comment (Newbrict) [inline simple_bloom_reachability_map.go]: Is there a reason to do it inline? personally I think it's easier to read through when it's like this, is inline more widely used ?

Comment (c00w) [inline simple_bloom_reachability_map.go]: No huge preference for inline vs not. I was thinking we would just evolve
this one.

**Comment by Newbrict** [comment]
Fixed issues in comments, and merged.

**Pull request merged**